src/components/new-request/search.vue

```diff
@@ -93,9 +93,23 @@
 
       <!-- Conversion (aka Alteration) flow -->
       <template v-else-if="isConversion">
-        <EntityType v-if="getSearchBusiness && isAlterable" />
         <BusinessLookupFetch />
-        <CompanyType v-if="showCompanyTypeRadioButtons" />
+        <EntityType v-if="getSearchBusiness && isAlterable" cols="12" md="12" />
+        <CompanyType v-if="!!getSearchBusiness && !!getConversionType" />
+
+        <template v-if="isNamedCompany">
+          <v-col cols="12" :md="showDesignation ? '8' : '12'">
+            <NameInput
+              :is-mras-search="(isXproFlow && isMrasJurisdiction && !getHasNoCorpNum)"
+              @emit-corp-num-validity="corpNumValid = $event"
+            />
+          </v-col>
+          <Designation v-if="showDesignation" cols="12" md="4" />
+        </template>
+
+        <template v-if="isNumberedCompany">
+          <NumberedCompanyBullets />
+        </template>
```

Comment:
I suggest moving the v-if into line 111 and getting rid of the template, but this isn't a deal-breaker.